core/src/main/kotlin/com/trevjonez/composer/ComposerParams.kt

```diff
@@ -44,7 +45,8 @@ data class ComposerParams(
                 "--test-apk", testApk.absolutePath)
                 .let { params ->
                   withOrchestrator?.takeIf { it }?.let {
-                    params + arrayOf("--with-orchestrator")
+                    val apks = orchestratorApks.map { file -> file.absolutePath }.toTypedArray()
+                    params + arrayOf("--with-orchestrator", "--orchestrator-apks", *apks)
```

Comment:
I wonder if it should just be "additional apk's" so that you could use it to facilitate something like "buy/install this other app to add features" in the end composer doesn't care what the extra apk is for right?